barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.test.cpp

```diff
@@ -66,16 +63,10 @@ TEST_F(AcirFormatTests, TestASingleConstraintNoPubInputs)
         .original_opcode_indices = create_empty_original_opcode_indices(),
     };
     mock_opcode_indices(constraint_system);
-    WitnessVector witness{ 0, 0, 1 };
+    WitnessVector witness{ 5, 7, 12 };
```

Comment:
Curious why the change?

Reply:
The new numbers are arbitrary but the old constraint simply wasnt valid. I think the only reason it was passing before was because of a quirk of the plonk pathway that adds an additional zero witness which offset these indices and made the constraint work out